client/src/components/IconTooltip/index.tsx

```diff
@@ -0,0 +1,136 @@
+import React from "react";
+import RightBoat from "../../assets/icons/boat right.png";
+import Volcano from "../../assets/icons/volcano.png";
+import Factory from "../../assets/icons/factory.png";
+import Deciduous from "../../assets/icons/deciduous.png";
+import DeciduousMultiple from "../../assets/icons/deciduousMultiples.png";
+import Evergreen from "../../assets/icons/evergreen.png";
+import EvergreenMultiple from "../../assets/icons/evergreenMultiples.png";
+import Pond from "../../assets/icons/pond.png";
+import PondMultiple from "../../assets/icons/pondMultiples.png";
+import Rock from "../../assets/icons/rock.png";
+import RockMultiple from "../../assets/icons/rockMultiples.png";
+import styles from "./styles.module.css";
+import styled from "styled-components";
+
+interface IconToolTipProps {
+  type:
+    | "string"
+    | "stringMultiple"
```

Comment:
BTW what does `stringMultiple` mean?

Reply:
I just meant it as Array<String>, List<String>, etc! Thought it might be nice to have more variety, but we don't need to keep it :P

Reply:
Since an array/list is just an object, maybe we can simply call it an `object`? The problem is that it is ambiguous. Also, how do we describe `List<List<String>>`? `stringMultipleMultiple` seems a bit crazy.